src/app/components/MediaLoader/types.ts

```diff
@@ -166,7 +167,45 @@ export type ClipMediaBlock = {
   };
 };
 
-export type MediaBlock = AresMediaBlock | ClipMediaBlock | CaptionBlock;
+export type TvMediaBlock = {
+  id: string;
+  subType: 'episode';
+  format: 'Video';
+  title: string;
+  synopses: {
+    short: string;
+    medium: string;
+  };
+  imageUrl: string;
+  embedding: boolean;
+  advertising: boolean;
+  versions: [
+    {
+      versionId: string;
+      types: [string];
```

Comment:
I think this needs to be 
```suggestion
      types: string[];
```